canonical-web-and-design/ubuntu.com
# Redirect ubuntu release notes

## Done

- Redirect ubuntu release notes - mostly
- **Note there are some special cases which might need to be added to this**

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: 
    - http://0.0.0.0:8001/GetUbuntu/releasenotes?os=ubuntukylin&ver=15.04&lang=zh
    - http://0.0.0.0:8001/GetUbuntu/releasenotes?os=ubuntu&ver=16.04
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that you mostly get to the right place


## Issue / Card

Fixes #7129
